frontend/micro-ui/web/micro-ui-internals/packages/Mukta/src/configs/UICustomizations.js

```diff
@@ -70,7 +70,7 @@ export const UICustomizations = {
           return <span>{t(`WF_EST_${value}`)}</span>;
 
          case "WORKS_ESTIMATED_AMOUNT":
-          return <span>{value ? Digit.Utils.dss.formatterWithoutRound(value, "number") : t("ES_COMMON_NA")}</span>;
+          return <p style={{ textAlign: 'right'}}>{value ? `${Digit.Utils.dss.formatterWithoutRound(value, "number")}` : t("ES_COMMON_NA")}</p>;
```

Comment:
@prachirane-eGov  can you put this as a common component for viewing any amount 
in to react-components and use it here

Reply:
Added Amount Component for this change.